### ISSUE TITLE
align_right_cmt_at_col >= 2, with no effect 

### ISSUE THREAD

**Issue by tyralek** [uncrustify/uncrustify#2264]
Uncrustify-0.68.1 under windows, executed from gitbash.

[Examples](https://github.com/tyralek/uncrustify_test)

In my configuration I have `align_right_cmt_at_col = 2`
but I have a lot other options set.

I expect to get:

```cpp
class Example :
    public Base1,
    public Base2
{
public:
    /// Example enum description.
    enum ExampleEnum
    {
        EXAMPLE_ENUM_FIRST = 0,  ///< First enum meaningful description.
        EXAMPLE_ENUM_SECOND = 4, ///< If you want to add comment on the side,
                                 ///< then remember about "<" character.
        EXAMPLE_ENUM_THIRD = 10, ///< As you can see above, multiline is also possible.
    };
```

But I have:
```cpp
class Example :
    public Base1,
    public Base2
{
public:
    /// Example enum description.
    enum ExampleEnum
    {
        EXAMPLE_ENUM_FIRST = 0, ///< First enum meaningful description.
        EXAMPLE_ENUM_SECOND = 4, ///< If you want to add comment on the side,
        ///< then remember about "<" character. 
        EXAMPLE_ENUM_THIRD = 10, ///< As you can see above, multiline is also possible.
    };
```

Please advise.
Lukasz

**Comment by mihaipopescu** [comment]
It may be linked with the infamous rule 3 of comment column alignment also discussed in other issues like this one #1134.

**Comment by tyralek** [comment]
The issue #1134 were closed on  on Jun 14, 2017
I am using uncrustify-0.68.1-win64 version released in Dec 2018.

Should I have it fixed in latest version?
Or It make sense to rebuild latest git sources?
Lukasz

**Comment by guy-maurel** [comment]
It always make sense to use the latest git sources.
Please test it and report.

**Comment by tyralek** [comment]
I have verified it with latest available binary v0.69, still issue exist for me.

**Comment by gmaurel** [comment]
A closing brace is missing at the end of your example.
I get:
```
Uncrustify_d-0.70.1-309-0745cc93
Parsing: tyralek.cpp as language CPP
//example file
class Example :
        public Base1,
        public Base2
{
public:
/// Example enum description.
enum ExampleEnum
{
        EXAMPLE_ENUM_FIRST = 0,  ///< First enum meaningful description.
        EXAMPLE_ENUM_SECOND = 4, ///< If you want to add comment on the side,
                                 ///< then remember about "<" character.
        EXAMPLE_ENUM_THIRD = 10, ///< As you can see above, multiline is also possible.
};
}

The current git version works pretty fine under Linux Mint.
  https://github.com/uncrustify/uncrustify
Please test it and report!
```
